change/office-ui-fabric-react-2020-06-15-13-23-43-jslone-sass-reference-build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "_common.scss: Update import statement to not depend on dist folder (which doesn't exist until full build)",
```

Comment:
@khmakoto let me know if this change description is OK. Happy to update it if it needs to be more clear.